omod-common/src/main/java/org/openmrs/module/webservices/docs/swagger/SwaggerSpecificationCreator.java

```diff
@@ -1069,7 +1069,10 @@ private Operation createOperation(DelegatingResourceHandler<?> resourceHandler,
 			
 			operation.setSummary("Fetch all non-retired");
 			operation.setOperationId("getAll" + getOperationTitle(resourceHandler, true));
-			operation.addResponse("200", response200.schema(new RefProperty("#/definitions/FetchAll")));
+			operation.addResponse("200",
+					response200.schema(new ArrayProperty(
+						new RefProperty(getSchemaRef(resourceName, resourceParentName, OperationEnum.get)))));
```

Comment:
I'm not deeply familiar with Swagger. Could you leave a comment with a brief description of how this changes things? Is it replacing the definition of the return value with something that matches the `GET` request?

Reply:
Currently, the `getAll` operation on a resouce is not properly typed in swagger. It returns a generic type called `FetchAll`, which has no typing information of the actual array being returned. (For example, `getAll` on patients should return `PatientGet[]`). This change should fix that.

Here's the typing for `getAll` on Patient currently:
![image](https://github.com/user-attachments/assets/f3361161-af5f-435c-adef-431f2e42ecea)